<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
    <meta charset="utf-8">
    <meta name="generator" content="quarto-1.5.24">

    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

    <meta name="author" content="Andreas Ludvig Ohm Svendsen">
    <meta name="author" content="Erkka Järvinen">
    <meta name="author" content="Tore B. Stage">
    <meta name="dcterms.date" content="2024-03-15">
    <meta name="keywords" content="3D primary human hepatocytes, Inflammation, Drug metabolism">

    <title>Inflammation and spheroids</title>
    <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      div.columns{display: flex; gap: min(4vw, 1.5em);}
      div.column{flex: auto; overflow-x: auto;}
      div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
      ul.task-list{list-style: none;}
      ul.task-list li input[type="checkbox"] {
        width: 0.8em;
        margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
        vertical-align: middle;
      }
      /* CSS for syntax highlighting */
      pre > code.sourceCode { white-space: pre; position: relative; }
      pre > code.sourceCode > span { line-height: 1.25; }
      pre > code.sourceCode > span:empty { height: 1.2em; }
      .sourceCode { overflow: visible; }
      code.sourceCode > span { color: inherit; text-decoration: inherit; }
      div.sourceCode { margin: 1em 0; }
      pre.sourceCode { margin: 0; }
      @media screen {
      div.sourceCode { overflow: auto; }
      }
      @media print {
      pre > code.sourceCode { white-space: pre-wrap; }
      pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
      }
      pre.numberSource code
        { counter-reset: source-line 0; }
      pre.numberSource code > span
        { position: relative; left: -4em; counter-increment: source-line; }
      pre.numberSource code > span > a:first-child::before
        { content: counter(source-line);
          position: relative; left: -1em; text-align: right; vertical-align: baseline;
          border: none; display: inline-block;
          -webkit-touch-callout: none; -webkit-user-select: none;
          -khtml-user-select: none; -moz-user-select: none;
          -ms-user-select: none; user-select: none;
          padding: 0 4px; width: 4em;
        }
      pre.numberSource { margin-left: 3em;  padding-left: 4px; }
      div.sourceCode
        {   }
      @media screen {
      pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
      }
      /* CSS for citations */
      div.csl-bib-body { }
      div.csl-entry {
        clear: both;
      }
      .hanging-indent div.csl-entry {
        margin-left:2em;
        text-indent:-2em;
      }
      div.csl-left-margin {
        min-width:2em;
        float:left;
      }
      div.csl-right-inline {
        margin-left:2em;
        padding-left:1em;
      }
      div.csl-indent {
        margin-left: 2em;
      }    </style>

    <style>
      body.hypothesis-enabled #quarto-embed-header {
        padding-right: 36px;
      }

      #quarto-embed-header {
        height: 3em;
        width: 100%;
        display: flex;
        justify-content: space-between;
        align-items: center;
        border-bottom: solid 1px;
      }

      #quarto-embed-header h6 {
        font-size: 1.1em;
        padding-top: 0.6em;
        margin-left: 1em;
        margin-right: 1em;
        font-weight: 400;
      }

      #quarto-embed-header a.quarto-back-link,
      #quarto-embed-header a.quarto-download-embed {
        font-size: 0.8em;
        margin-top: 1em;
        margin-bottom: 1em;
        margin-left: 1em;
        margin-right: 1em;
      }

      .quarto-back-container {
        padding-left: 0.5em;
        display: flex;
      }

      .headroom {
          will-change: transform;
          transition: transform 200ms linear;
      }

      .headroom--pinned {
          transform: translateY(0%);
      }

      .headroom--unpinned {
          transform: translateY(-100%);
      }      
    </style>

    <script>
    window.document.addEventListener("DOMContentLoaded", function () {

      var header = window.document.querySelector("#quarto-embed-header");
      const titleBannerEl = window.document.querySelector("body > #title-block-header");
      if (titleBannerEl) {
        titleBannerEl.style.paddingTop = header.clientHeight + "px";
      }
      const contentEl = window.document.getElementById('quarto-content');
      for (const child of contentEl.children) {
        child.style.paddingTop = header.clientHeight + "px";
        child.style.marginTop = "1em";
      }

      // Use the article root if the `back` call doesn't work. This isn't perfect
      // but should typically work
      window.quartoBackToArticle = () => {
        var currentUrl = window.location.href;
        window.history.back();
        setTimeout(() => {
            // if location was not changed in 100 ms, then there is no history back
            if(currentUrl === window.location.href){              
                // redirect to site root
                window.location.href = "index.html";
            }
        }, 100);
      }

      const headroom = new window.Headroom(header, {
        tolerance: 5,
        onPin: function () {
        },
        onUnpin: function () {
        },
      });
      headroom.init();
    });
    </script>

    
<script src="site_libs/manuscript-notebook/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
     <script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>  
      <meta name="citation_title" content="Inflammation and spheroids">
<meta name="citation_abstract" content="This is an abstract ...
">
<meta name="citation_keywords" content="3D primary human hepatocytes,Inflammation,Drug metabolism">
<meta name="citation_author" content="Andreas Ludvig Ohm Svendsen">
<meta name="citation_author" content="Erkka Järvinen">
<meta name="citation_author" content="Tore B. Stage">
<meta name="citation_publication_date" content="2024-03-15">
<meta name="citation_cover_date" content="2024-03-15">
<meta name="citation_year" content="2024">
<meta name="citation_online_date" content="2024-03-15">
<meta name="citation_language" content="en">
<meta name="citation_reference" content="citation_title=Strategies for the development of volcanic hazard maps in monogenetic volcanic fields: The example of La Palma (Canary Islands);,citation_author=José Marrero;,citation_author=Alicia García;,citation_author=Manuel Berrocoso;,citation_author=Ángeles Llinares;,citation_author=Antonio Rodríguez-Losada;,citation_author=R. Ortiz;,citation_publication_date=2019-07;,citation_cover_date=2019-07;,citation_year=2019;,citation_doi=10.1186/s13617-019-0085-5;,citation_volume=8;,citation_journal_title=Journal of Applied Volcanology;">
<meta name="citation_reference" content="citation_title=Clinical and Molecular Perspectives on Inflammation-Mediated Regulation of Drug Metabolism and Transport;,citation_author=Ann-Cathrine Dalgård Dunvald;,citation_author=Erkka Järvinen;,citation_author=Christina Mortensen;,citation_author=Tore B. Stage;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/cpt.2432;,citation_issue=n/a;,citation_doi=10.1002/cpt.2432;,citation_volume=n/a;,citation_language=en;,citation_journal_title=Clinical Pharmacology &amp;amp;amp; Therapeutics;">
<meta name="citation_reference" content="citation_title=Serum cytokine profiles in healthy young and elderly population assessed using multiplexed bead-based immunoassays;,citation_author=Hyun Ok Kim;,citation_author=Han-Soo Kim;,citation_author=Jong-Chan Youn;,citation_author=Eui-Cheol Shin;,citation_author=Sungha Park;,citation_publication_date=2011-07-20;,citation_cover_date=2011-07-20;,citation_year=2011;,citation_fulltext_html_url=https://doi.org/10.1186/1479-5876-9-113;,citation_issue=1;,citation_doi=10.1186/1479-5876-9-113;,citation_volume=9;,citation_journal_title=Journal of Translational Medicine;">
<meta name="citation_reference" content="citation_title=Cytokine Levels in the Serum of Healthy Subjects;,citation_author=Giulio Kleiner;,citation_author=Annalisa Marcuzzi;,citation_author=Valentina Zanin;,citation_author=Lorenzo Monasta;,citation_author=Giorgio Zauli;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_fulltext_html_url=http://www.hindawi.com/journals/mi/2013/434010/;,citation_doi=10.1155/2013/434010;,citation_volume=2013;,citation_language=en;,citation_journal_title=Mediators of Inflammation;">
<meta name="citation_reference" content="citation_title=MIQE &amp;amp;amp; qPCR ibook how to apply the MIQE guidelines - a visual, interactive and practical qPCR guide!;,citation_author=Afif M Abdel Nour;,citation_author=Michael W Pfaffl;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=https://itunes.apple.com/us/book/miqe-qpcr/id993276375?mt=11;,citation_language=en;">
<meta name="citation_reference" content="citation_title=Defining IL-6 levels in healthy individuals: A meta-analysis;,citation_author=Elias A. Said;,citation_author=Iman Al-Reesi;,citation_author=Nawal Al-Shizawi;,citation_author=Sanjay Jaju;,citation_author=Mohammed S. Al-Balushi;,citation_author=Crystal Y. Koh;,citation_author=Ali A. Al-Jabri;,citation_author=Lakshmanan Jeyaseelan;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.26654;,citation_issue=6;,citation_doi=10.1002/jmv.26654;,citation_volume=93;,citation_language=en;,citation_journal_title=Journal of Medical Virology;">
<meta name="citation_reference" content="citation_title=High levels of interleukin-6 in patients with rheumatoid arthritis are associated with greater improvements in health-related quality of life for sarilumab compared with adalimumab;,citation_author=Vibeke Strand;,citation_author=Susan H. Boklage;,citation_author=Toshio Kimura;,citation_author=Florence Joly;,citation_author=Anita Boyapati;,citation_author=Jérôme Msihid;,citation_publication_date=2020-10-20;,citation_cover_date=2020-10-20;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1186/s13075-020-02344-3;,citation_issue=1;,citation_doi=10.1186/s13075-020-02344-3;,citation_volume=22;,citation_journal_title=Arthritis Research &amp;amp;amp; Therapy;">
<meta name="citation_reference" content="citation_title=Effect of Proinflammatory Cytokines (IL-6, TNF-&amp;amp;amp;lt;i&amp;gt;α&amp;lt;/i&amp;gt;, and IL-1<i>β</i>) on Clinical Manifestations in Indian SLE Patients;,citation_author=Vinod Umare;,citation_author=Vandana Pradhan;,citation_author=Milind Nadkar;,citation_author=Anjali Rajadhyaksha;,citation_author=Manisha Patwardhan;,citation_author=Kanjaksha K. Ghosh;,citation_author=Anita H. Nadkarni;,citation_publication_date=2014-12-07;,citation_cover_date=2014-12-07;,citation_year=2014;,citation_fulltext_html_url=https://www.hindawi.com/journals/mi/2014/385297/;,citation_doi=10.1155/2014/385297;,citation_volume=2014;,citation_language=en;,citation_journal_title=Mediators of Inflammation;">
<meta name="citation_reference" content="citation_title=Plasma interleukin-6 concentration for the diagnosis of sepsis in critically ill adults;,citation_author=Daniel Molano Franco;,citation_author=Ingrid Arevalo-Rodriguez;,citation_author=Marta Roqué Figuls;,citation_author=Nadia G. Montero Oleas;,citation_author=Xavier Nuvials;,citation_author=Javier Zamora;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://www.cochranelibrary.com/cdsr/doi/10.1002/14651858.CD011811.pub2/full;,citation_issue=4;,citation_doi=10.1002/14651858.CD011811.pub2;,citation_language=en;,citation_journal_title=Cochrane Database of Systematic Reviews;">
<meta name="citation_reference" content="citation_title=Characterization of primary human hepatocyte spheroids as a model system for drug-induced liver injury, liver function and disease;,citation_author=Catherine C. Bell;,citation_author=Delilah F. G. Hendriks;,citation_author=Sabrina M. L. Moro;,citation_author=Ewa Ellis;,citation_author=Joanne Walsh;,citation_author=Anna Renblom;,citation_author=Lisa Fredriksson Puigvert;,citation_author=Anita C. A. Dankers;,citation_author=Frank Jacobs;,citation_author=Jan Snoeys;,citation_author=Rowena L. Sison-Young;,citation_author=Rosalind E. Jenkins;,citation_author=Åsa Nordling;,citation_author=Souren Mkrtchian;,citation_author=B. Kevin Park;,citation_author=Neil R. Kitteringham;,citation_author=Christopher E. P. Goldring;,citation_author=Volker M. Lauschke;,citation_author=Magnus Ingelman-Sundberg;,citation_publication_date=2016-05;,citation_cover_date=2016-05;,citation_year=2016;,citation_doi=10.1038/srep25187;,citation_volume=6;,citation_language=eng;,citation_journal_title=Scientific Reports;">
<meta name="citation_reference" content="citation_title=Comparison of Hepatic 2D Sandwich Cultures and 3D Spheroids for Long-term Toxicity Applications: A Multicenter Study;,citation_author=Catherine C. Bell;,citation_author=Anita C. A. Dankers;,citation_author=Volker M. Lauschke;,citation_author=Rowena Sison-Young;,citation_author=Roz Jenkins;,citation_author=Cliff Rowe;,citation_author=Chris E. Goldring;,citation_author=Kevin Park;,citation_author=Sophie L. Regan;,citation_author=Tracy Walker;,citation_author=Chris Schofield;,citation_author=Audrey Baze;,citation_author=Alison J. Foster;,citation_author=Dominic P. Williams;,citation_author=Amy W. M. Ven;,citation_author=Frank Jacobs;,citation_author=Jos Houdt;,citation_author=Tuula Lähteenmäki;,citation_author=Jan Snoeys;,citation_author=Satu Juhila;,citation_author=Lysiane Richert;,citation_author=Magnus Ingelman-Sundberg;,citation_publication_date=2018-04-01;,citation_cover_date=2018-04-01;,citation_year=2018;,citation_issue=2;,citation_doi=10.1093/toxsci/kfx289;,citation_volume=162;,citation_language=eng;,citation_journal_title=Toxicological Sciences: An Official Journal of the Society of Toxicology;">
<meta name="citation_reference" content="citation_title=3D Spheroid Primary Human Hepatocytes for Prediction of Cytochrome P450 and Drug Transporter Induction;,citation_author=Erkka Järvinen;,citation_author=Helen S. Hammer;,citation_author=Oliver Pötz;,citation_author=Magnus Ingelman-Sundberg;,citation_author=Tore Bjerregaard Stage;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/cpt.2887;,citation_issue=6;,citation_doi=10.1002/cpt.2887;,citation_volume=113;,citation_language=en;,citation_journal_title=Clinical Pharmacology &amp;amp;amp; Therapeutics;">
<meta name="citation_reference" content="citation_title=3D human liver spheroids for translational pharmacology and toxicology;,citation_author=Magnus Ingelman-Sundberg;,citation_author=Volker M. Lauschke;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/bcpt.13587;,citation_issue=n/a;,citation_doi=10.1111/bcpt.13587;,citation_volume=n/a;,citation_language=en;,citation_journal_title=Basic &amp;amp;amp; Clinical Pharmacology &amp; Toxicology;">
<meta name="citation_reference" content="citation_title=Characterization of primary human hepatocyte spheroids as a model system for drug-induced liver injury, liver function and disease;,citation_author=Catherine C. Bell;,citation_author=Delilah F. G. Hendriks;,citation_author=Sabrina M. L. Moro;,citation_author=Ewa Ellis;,citation_author=Joanne Walsh;,citation_author=Anna Renblom;,citation_author=Lisa Fredriksson Puigvert;,citation_author=Anita C. A. Dankers;,citation_author=Frank Jacobs;,citation_author=Jan Snoeys;,citation_author=Rowena L. Sison-Young;,citation_author=Rosalind E. Jenkins;,citation_author=Åsa Nordling;,citation_author=Souren Mkrtchian;,citation_author=B. Kevin Park;,citation_author=Neil R. Kitteringham;,citation_author=Christopher E. P. Goldring;,citation_author=Volker M. Lauschke;,citation_author=Magnus Ingelman-Sundberg;,citation_publication_date=2016-05;,citation_cover_date=2016-05;,citation_year=2016;,citation_doi=10.1038/srep25187;,citation_volume=6;,citation_language=eng;,citation_journal_title=Scientific Reports;">
<meta name="citation_reference" content="citation_title=Characterization of primary human hepatocyte spheroids as a model system for drug-induced liver injury, liver function and disease;,citation_author=Catherine C. Bell;,citation_author=Delilah F. G. Hendriks;,citation_author=Sabrina M. L. Moro;,citation_author=Ewa Ellis;,citation_author=Joanne Walsh;,citation_author=Anna Renblom;,citation_author=Lisa Fredriksson Puigvert;,citation_author=Anita C. A. Dankers;,citation_author=Frank Jacobs;,citation_author=Jan Snoeys;,citation_author=Rowena L. Sison-Young;,citation_author=Rosalind E. Jenkins;,citation_author=Åsa Nordling;,citation_author=Souren Mkrtchian;,citation_author=B. Kevin Park;,citation_author=Neil R. Kitteringham;,citation_author=Christopher E. P. Goldring;,citation_author=Volker M. Lauschke;,citation_author=Magnus Ingelman-Sundberg;,citation_publication_date=2016-05;,citation_cover_date=2016-05;,citation_year=2016;,citation_doi=10.1038/srep25187;,citation_volume=6;,citation_language=eng;,citation_journal_title=Scientific Reports;">
<meta name="citation_reference" content="citation_title=Optimization of phenol-chloroform RNA extraction;,citation_author=Lee S. Toni;,citation_author=Anastacia M. Garcia;,citation_author=Danielle A. Jeffrey;,citation_author=Xuan Jiang;,citation_author=Brian L. Stauffer;,citation_author=Shelley D. Miyamoto;,citation_author=Carmen C. Sucharov;,citation_publication_date=2018-01-01;,citation_cover_date=2018-01-01;,citation_year=2018;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S2215016118300773;,citation_doi=10.1016/j.mex.2018.05.011;,citation_volume=5;,citation_journal_title=MethodsX;">
<meta name="citation_reference" content="citation_title=How to do successful gene expression analysis using real-time PCR;,citation_author=Stefaan Derveaux;,citation_author=Jo Vandesompele;,citation_author=Jan Hellemans;,citation_publication_date=2010-04-01;,citation_cover_date=2010-04-01;,citation_year=2010;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S1046202309002461;,citation_issue=4;,citation_doi=10.1016/j.ymeth.2009.11.001;,citation_volume=50;,citation_journal_title=Methods;,citation_series_title=The ongoing evolution of qPCR;">
<meta name="citation_reference" content="citation_title=Analysis of Relative Gene Expression Data Using Real-Time Quantitative PCR and the 2-ΔΔCT Method;,citation_author=Kenneth J. Livak;,citation_author=Thomas D. Schmittgen;,citation_publication_date=2001-12-01;,citation_cover_date=2001-12-01;,citation_year=2001;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S1046202301912629;,citation_issue=4;,citation_doi=10.1006/meth.2001.1262;,citation_volume=25;,citation_language=en;,citation_journal_title=Methods;">
<meta name="citation_reference" content="citation_title=The Ultimate qPCR Experiment: Producing Publication Quality, Reproducible Data the First Time;,citation_author=Sean C. Taylor;,citation_author=Katia Nadeau;,citation_author=Meysam Abbasi;,citation_author=Claude Lachance;,citation_author=Marie Nguyen;,citation_author=Joshua Fenrich;,citation_publication_date=2019-07;,citation_cover_date=2019-07;,citation_year=2019;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0167779918303421;,citation_issue=7;,citation_doi=10.1016/j.tibtech.2018.12.002;,citation_volume=37;,citation_language=en;,citation_journal_title=Trends in Biotechnology;">
<meta name="citation_reference" content="citation_title=What exactly is ’N’ in cell culture and animal experiments?;,citation_author=Stanley E. Lazic;,citation_author=Charlie J. Clarke-Williams;,citation_author=Marcus R. Munafò;,citation_publication_date=2018-04;,citation_cover_date=2018-04;,citation_year=2018;,citation_issue=4;,citation_doi=10.1371/journal.pbio.2005282;,citation_volume=16;,citation_language=eng;,citation_journal_title=PLoS biology;">
<meta name="citation_reference" content="citation_title=Direct Quantification of Cytochromes P450 and Drug Transporters-A Rapid, Targeted Mass Spectrometry-Based Immunoassay Panel for Tissues and Cell Culture Lysates;,citation_author=Frederik Weiß;,citation_author=Helen S. Hammer;,citation_author=Kathrin Klein;,citation_author=Hannes Planatscher;,citation_author=Ulrich M. Zanger;,citation_author=Agneta Norén;,citation_author=Christine Wegler;,citation_author=Per Artursson;,citation_author=Thomas O. Joos;,citation_author=Oliver Poetz;,citation_publication_date=2018-04;,citation_cover_date=2018-04;,citation_year=2018;,citation_issue=4;,citation_doi=10.1124/dmd.117.078626;,citation_volume=46;,citation_language=eng;,citation_journal_title=Drug Metabolism and Disposition: The Biological Fate of Chemicals;">
<meta name="citation_reference" content="citation_title=Enrichment-free High-throughput Liquid ChromatographyMultiple-Reaction Monitoring Quantification of Cytochrome P450 Proteins in Plated Human Hepatocytes Direct from 96-Well Plates Enables Routine Protein Induction Measurements;,citation_author=John P. Savaryn;,citation_author=Ning Liu;,citation_author=Jun Sun;,citation_author=Junli Ma;,citation_author=David M. Stresser;,citation_author=Gary Jenkins;,citation_publication_date=2020-07-01;,citation_cover_date=2020-07-01;,citation_year=2020;,citation_fulltext_html_url=https://dmd.aspetjournals.org/content/48/7/594;,citation_issue=7;,citation_doi=10.1124/dmd.120.090480;,citation_volume=48;,citation_language=en;,citation_journal_title=Drug Metabolism and Disposition;">
</head>

  <body class="quarto-notebook">
    <div id="quarto-embed-header" class="headroom fixed-top bg-primary">
      
      <a onclick="window.quartoBackToArticle(); return false;" class="btn btn-primary quarto-back-link" href=""><i class="bi bi-caret-left"></i> Back to Article</a>
      <h6><i class="bi bi-journal-code"></i> Article Notebook</h6>

            <a href="./index.qmd" class="btn btn-primary quarto-download-embed" download="index.qmd">Download Source</a>
          </div>

     <header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Inflammation and spheroids</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Andreas Ludvig Ohm Svendsen <a href="mailto:alosvendsen@health.sdu.dk" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-8050-6447" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        SDU
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Erkka Järvinen </p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        SDU?
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Tore B. Stage </p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        SDU
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
                <div>
            <div class="quarto-title-meta-heading">Published</div>
            <div class="quarto-title-meta-contents">
              <p class="date">March 15, 2024</p>
            </div>
          </div>
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      </div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>This is an abstract …</p>
      </div>
    </div>

    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>3D primary human hepatocytes, Inflammation, Drug metabolism</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    <div class="quarto-code-links"><div class="quarto-title-meta-heading">Code Links</div><div class="quarto-title-meta-contents"><span><a href="https://github.com/andreasludvig/inflammation_spheroids/" target="_blank"><i class="bi bi-github"></i>GitHub Repo</a></span></div></div></div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods">Methods</a>
  <ul class="collapse">
  <li><a href="#d-spheroid-culture-of-phhs" id="toc-d-spheroid-culture-of-phhs" class="nav-link" data-scroll-target="#d-spheroid-culture-of-phhs">3D spheroid culture of PHHs</a></li>
  <li><a href="#treatments" id="toc-treatments" class="nav-link" data-scroll-target="#treatments">Treatments</a></li>
  <li><a href="#spheroid-viability-and-morphology" id="toc-spheroid-viability-and-morphology" class="nav-link" data-scroll-target="#spheroid-viability-and-morphology">Spheroid viability and morphology</a></li>
  <li><a href="#rna-extraction-cdna-synthesis-and-qpcr" id="toc-rna-extraction-cdna-synthesis-and-qpcr" class="nav-link" data-scroll-target="#rna-extraction-cdna-synthesis-and-qpcr">RNA extraction, cDNA synthesis and qPCR</a></li>
  <li><a href="#activity" id="toc-activity" class="nav-link" data-scroll-target="#activity">Activity</a></li>
  <li><a href="#quantitative-analysis-of-proteins-via-lc-ms-with-selective-immunoprecipitation" id="toc-quantitative-analysis-of-proteins-via-lc-ms-with-selective-immunoprecipitation" class="nav-link" data-scroll-target="#quantitative-analysis-of-proteins-via-lc-ms-with-selective-immunoprecipitation">Quantitative analysis of proteins via LC-MS with selective immunoprecipitation</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#viability-and-morphology" id="toc-viability-and-morphology" class="nav-link" data-scroll-target="#viability-and-morphology">Viability and morphology</a></li>
  <li><a href="#effect-of-inflammatory-cytokines-on-mrna-expression" id="toc-effect-of-inflammatory-cytokines-on-mrna-expression" class="nav-link" data-scroll-target="#effect-of-inflammatory-cytokines-on-mrna-expression">Effect of inflammatory cytokines on mRNA expression</a></li>
  </ul></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">      

       <div class="cell-container"><div class="cell-decorator"><pre>In [1]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Always load same package versions</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>groundhog.day <span class="ot">&lt;-</span> <span class="st">"2024-01-01"</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>groundhog<span class="sc">::</span><span class="fu">meta.groundhog</span>(groundhog.day) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>[1;36mNow using 'groundhog_3.1.2'.[0m</code></pre>
</div>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>pkgs <span class="ot">&lt;-</span>  <span class="fu">c</span>(<span class="st">"data.table"</span>)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="fu">groundhog.library</span>(pkgs, groundhog.day)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>[36mSuccessfully attached 'data.table_1.14.10'[0m</code></pre>
</div>
</div></div>
<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>Inflammation is a complex biological response that is pivotal in various pathological conditions. These range from systemic inflammatory diseases, such as rheumatoid arthritis and sepsis, to lower-grade chronic inflammatory states such as type 2 diabetes mellitus. Given the prevalence of systemic inflammation, understanding its interaction with drug metabolism is of substantial clinical relevance.</p>
<p>Drug-metabolizing enzymes and transporters (DMETs), predominantly found in hepatocytes within the liver, are central to the biotransformation of a wide variety of compounds. Inflammation has been shown to modulate the activity of these DMETs, a phenomenon that could potentially affect the pharmacokinetics of numerous medications. For individuals with altered inflammatory status—whether due to a chronic condition like diabetes or an acute event like sepsis—this modulation can have significant implications. It may necessitate adjustments in drug dosages to avoid adverse effects or therapeutic failure.</p>
<p>Previous research has provided valuable insights into the effects of inflammation on DMETs, but a clear correlation between in vitro studies and clinical observations remains elusive. For instance Dunvald et al. <span class="citation" data-cites="dunvald">(<a href="#ref-dunvald" role="doc-biblioref">1</a>)</span> conducted a comprehensive review of the clinical and in vitro evidence on inflammation-mediated modulation of DMETs and the impact on drug metabolism in humans. They found that in vitro studies in primary human hepatocytes revealed strong evidence of downregulation of key cytochrome P450 (CYP) enzymes by inflammatory cytokines such as IL-6 and IL-1β. However, these studies often employed supraphysiological cytokine doses, which may not accurately represent the inflammatory conditions observed in patients.</p>
<p>Levels of IL-6 and IL-1β in healthy individuals are generally low, with reports in range of ~10pg/ml in adults for IL-6, and ~2.5 pg/ml IL-1β in adults <span class="citation" data-cites="kim2011 kleiner2013 said2021 strand2020">(<a href="#ref-kim2011" role="doc-biblioref">2</a>–<a href="#ref-strand2020" role="doc-biblioref">5</a>)</span>. In contrast, cytokine levels may be considerably elevated with IL-6 levels of ~140 pg/mL, and IL-1β <u>levels of ~100 pg/mL</u>, among patients with rheumatoid arthritis or SLE <span class="citation" data-cites="umare2014">(<a href="#ref-umare2014" role="doc-biblioref">6</a>)</span> to more than 1 ng/mL of IL-6 for patients with acute inflammation caused by sepsis <span class="citation" data-cites="franco2019">(<a href="#ref-franco2019" role="doc-biblioref">7</a>)</span>. These variations in cytokine levels, which span a wide range in different pathological states, highlight the complex and dynamic nature of inflammation and underscore the need for research that considers this variability when investigating the effects of inflammation on drug-metabolizing enzymes.</p>
<p>The median IL-6 values (IQR) in the control, sepsis, and septic shock groups were 23.6 (11.2–43.5), 89.9 (45.2–272.6), and 1378.6 (256.4–11,062.1) pg/mL</p>
<p><a href="https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-019-4618-7" class="uri">https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-019-4618-7</a></p>
<p>Recently, 3D primary human hepatocytes (PHH) have challenged 2D PHH as a more physiologically relevant culture method of PHH. 3D culture leads to more stable cell cultures that retain their hepatic phenotype for extended periods of time <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">8</a>)</span>. Consequently, this 3D PHH have been shown to predict CYP induction and hepatotoxicity more accurately than 2D PHH <span class="citation" data-cites="bell2018 järvinen2023">(<a href="#ref-bell2018" role="doc-biblioref">9</a>,<a href="#ref-järvinen2023" role="doc-biblioref">10</a>)</span>. Historically, 2D PHH have been utilized to study the effect of drugs and inflammation on hepatocyte/liver function. However, the inherent limitations of 2D cultures, primarily their inability to maintain the physiological phenotype and liver-specific functions of hepatocytes, have prompted a shift towards the 3D liver spheroid models. This model is increasingly recognized for their physiological relevance and stability, offering a more accurate representation of hepatic responses. The 3D liver spheroids preserve liver cell phenotypes and functions over extended periods, thereby enhancing the reliability of drug-induced liver injury predictions and disease mechanism investigations. This advancement positions 3D liver spheroids as potentially the new standard for in vitro hepatocyte studies, while still being suitable for a high throughput setting, and financially accessible as opposed to even more advanced liver models (ref for last part) <span class="citation" data-cites="ingelman-sundberg dunvald">(<a href="#ref-dunvald" role="doc-biblioref">1</a>,<a href="#ref-ingelman-sundberg" role="doc-biblioref">11</a>)</span> . Another claim for the lack of correlation discussed in the review by AC et al.&nbsp;is that there might be methodological limitations to the widespread use of 2D models of PHHs <span class="citation" data-cites="dunvald">(<a href="#ref-dunvald" role="doc-biblioref">1</a>)</span>.</p>
<p>We aimed to utilize 3D primary human hepatocytes <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">8</a>)</span> to study the impact of physiologically relevant concentrations of cytokines on CYP expression and activity. This may help further our understanding of the impact of inflammation on clinical drug metabolism among patients with inflammation. This, in turn, may inform more precise and adaptive prescribing strategies for patients in various inflammatory states.</p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<section id="d-spheroid-culture-of-phhs" class="level3">
<h3 class="anchored" data-anchor-id="d-spheroid-culture-of-phhs">3D spheroid culture of PHHs</h3>
<section id="materials" class="level4">
<h4 class="anchored" data-anchor-id="materials">Materials</h4>
<p>All necessary components for cell culture, including the medium, supplements, and compounds, were acquired from Thermo Fisher Scientific, unless stated otherwise.</p>
</section>
<section id="cell-source-and-preparation" class="level4">
<h4 class="anchored" data-anchor-id="cell-source-and-preparation">Cell source and preparation</h4>
<p>Three lots of cryopreserved primary human hepatocytes, specifically Hu8345-A, Hu8373-A and Hu8381, were procured from Thermo Fisher Scientific (Waltham, MA). All necessary cell culture reagents were sourced from the same supplier. All PHHs were qualified for spheroid formation per the manufacturer.</p>
</section>
<section id="spheroid-formation-and-maintenance" class="level4">
<h4 class="anchored" data-anchor-id="spheroid-formation-and-maintenance">Spheroid formation and maintenance</h4>
<p>The spheroid culture followed an adapted protocol from a previous study <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">8</a>)</span>. In short 1500 hepatocytes per well in ultra-low attachment 96-well plates. Following seeding, the plates were centrifuged for 2 minutes at 200g. Subsequently, the plates were incubated at 37°C with 5% CO2.</p>
<p>On day 0 cells were seeded in culture medium, totaling 100 μL per well, consisting of 5% fetal bovine serum, 1 μM dexamethasone, 5 μg/mL human recombinant insulin, 100 U/mL penicillin, 100 μg/mL streptomycin, 2 mM L-glutamine, and 15 mM HEPES in Williams’ E medium.</p>
<p>After spheroid formation the seeding medium was replaced with FBS free maintenance medium. This medium comprised 0.1 μM dexamethasone, 10 μg/mL human recombinant insulin, 5.5 μg/mL transferrin, 6.7 ng/mL selenium, 100 U/mL penicillin, 100 μg/mL streptomycin, and 2 mM L-glutamine in Williams’ E medium. This FBS washout with maintenance medium was done with a 70% medium change on days 5-7. <code>For donor 1 (Hu8345-A) the maintenance also contained 5.35 μg/mL linoleic acid, 1.25 mg/mL bovine serum albumin and 15 mM HEPES. These supplements are, however, no longer recommended by the manufacturer</code><a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a><code>.</code></p>
<div class="no-row-height"><div id="fn1"><p><sup>1</sup>&nbsp;Skip this?</p></div></div><p>Detailed descriptions of the spheroid culture process, as well as donor information, are available in the supplementary materials (Table SX).</p>
</section>
</section>
<section id="treatments" class="level3">
<h3 class="anchored" data-anchor-id="treatments">Treatments</h3>
<p>Treatment with proinflammatory cytokines commenced on day 8 post seeding. The spheroids were exposed to IL-6 and IL-1β at concentrations of 0.01, 0.1, 1, and 10 ng/ml, alongside a vehicle control (0.1% bovine serum albumin in PBS). The cytokine treatments and vehicle control were diluted 1:1000 in the culture medium before administration.Re-dosing was done every other day for long-term cultures.</p>
</section>
<section id="spheroid-viability-and-morphology" class="level3">
<h3 class="anchored" data-anchor-id="spheroid-viability-and-morphology">Spheroid viability and morphology</h3>
<p>The viability of the spheroids was monitored by measuring their ATP content, using the luminiscence CellTiter-Glo 3D assay from Promega (Madison, WI). Quantification of ATP was done using an ATP (from Thermo Fisher Scientific) standard curve. Viability was assessed every 2-3 days throughout the experiments.</p>
<p>In addition to viability testing, the morphology of the spheroids was monitored. Imaging was done every second or third day, aligning with the viability assays. The spheroids were imaged using a light microscope equipped with a 10X magnification lens and a digital camera from Motic (Hong Kong, China).</p>
</section>
<section id="rna-extraction-cdna-synthesis-and-qpcr" class="level3">
<h3 class="anchored" data-anchor-id="rna-extraction-cdna-synthesis-and-qpcr">RNA extraction, cDNA synthesis and qPCR</h3>
<p>RNA was extracted using the phenol-chloroform method, following the manufacturer’s protocol from Qiagen with minor modifications <span class="citation" data-cites="toni2018">(<a href="#ref-toni2018" role="doc-biblioref">12</a>)</span>. Each spheroid pool was treated with 1 mL of Qiazol (Qiagen) reagent for lysis, followed by co-precipitation with 15 μg of RNA grade glycogen (Thermo Fisher Scientific). The RNA pellet underwent three washes with 75% ethanol. Post-wash, a combination of heating and evaporation techniques was applied to remove residual ethanol and to facilitate the solubilization of the RNA pellet in water.</p>
<p>RNA integrity was assessed for a subset of samples<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> using the Agilent BioAnalyzer, yielding RNA Integrity Number (RIN) values from 7 to 9.8. These values indicate high-quality RNA. Concentration of RNA were estimated utilizing the NanoDrop spectrophotometer.</p>
<div class="no-row-height"><div id="fn2"><p><sup>2</sup>&nbsp;Dog ikke fra nogle af disse 3 donorer, så måske bedre med: To assess the integrity of the RNA, we evaluated the RNA Integrity Number (RIN) for a subset of samples, which yielded values ranging from 7 to 9.8.<br>
<br>
To validate the RNA extraction method, we assessed the RNA Integrity Number (RIN) using the Agilent BioAnalyzer for a subset of samples not derived from the donors discussed in this article. These assessments yielded RIN values between 7 and 9.8, indicative of high-quality RNA.</p></div></div><p>Approximately 500 ng of extracted RNA was used for cDNA synthesis. This process was performed using the High-Capacity cDNA Reverse Transcription Kit with RNase Inhibitor (Thermo Fisher Scientific).</p>
<p>For qPCR, 10 ng of cDNA, based on the initial RNA concentration, was utilized for each reaction. The reactions were set up in a 10 μL volume containing TaqMan Universal Master Mix II and target-specific TaqMan assays (detailed in Supplementary Methods). The qPCR was run for 40 cycles, following the manufacturer’s guidelines (Thermo Fisher Scientific). The PCR plate sample maximization method was applied<span class="citation" data-cites="derveaux2010">(<a href="#ref-derveaux2010" role="doc-biblioref">13</a>)</span>, and a stable reference gene as well as no template controls and no reverse transcriptase controls was included in each plate.</p>
<p>In the analysis of mRNA expression, the relative quantity was calculated by first determining the delta Cq ((C_q)) for each sample. This was achieved by subtracting the mean Cq of the control group from the mean Cq of the replicates of each sample. The relative quantity was then computed using the formula (2^{C_q}). For normalization, the relative quantity of the target gene in each sample was divided by the relative quantity of the reference gene (GAPDH or TBP). The average normalized expression for each biological group was subsequently calculated.</p>
</section>
<section id="activity" class="level3">
<h3 class="anchored" data-anchor-id="activity">Activity</h3>
<p>To assess the activity of CYP3A4 (maybe otheres also), an experiment was conducted using midazolam as a substrate in spheroids derived from donors 2 and 3, evaluated on day 12. Midazolam (Toronto Research Chemicals, Toronto, ON, Canada), was prepared as a 6,000× stock solution in DMSO. The final DMSO concentration in the enzyme activity assays was maintained at 0.15%. Prior to the introduction of midazolam to the spheroids, the cells underwent three washes with 100 μL of maintenance medium. Between washes, the residual volume was kept at 20 μL, and before the final wash, the spheroids were incubated for 2 hours in a cell culture incubator.</p>
<p>The concentration of midazolam in the assay was set at 10 μM, with a total volume of 100 μL per well. For the analysis, triplicate samples, each comprising a <del>spheroid</del> and medium from a single well, were collected at 0.5 (8 hours for other enzymes) hours post midazolam treatment. These samples were then stored at -80°C until further analysis. The specific methods used for the metabolite analysis of these samples are detailed in the supplementary material.</p>
</section>
<section id="quantitative-analysis-of-proteins-via-lc-ms-with-selective-immunoprecipitation" class="level3">
<h3 class="anchored" data-anchor-id="quantitative-analysis-of-proteins-via-lc-ms-with-selective-immunoprecipitation">Quantitative analysis of proteins via LC-MS with selective immunoprecipitation</h3>
<p>The quantification of human CYP <del>and transporter</del> proteins was performed using a targeted liquid chromatography-mass spectrometry approach, incorporating selective immunoprecipitation of peptides, as previously described<span class="citation" data-cites="weiß2018">(<a href="#ref-weiß2018" role="doc-biblioref">14</a>)</span>. Uniform numbers of spheroids were employed in each analysis for consistency. In summary, spheroids were lysed in a buffer of 42 mM ammonium bicarbonate, containing 1.14 mM dithiothreitol and 9 mM iodoacetamide, and incubated for 15 minutes at 90°C, a method adapted from Savaryn et al. <span class="citation" data-cites="savaryn2020">(<a href="#ref-savaryn2020" role="doc-biblioref">15</a>)</span>. The lysates were then digested with MS-grade Pierce Trypsin Protease (Thermo Fisher Scientific) for 16 hours at 37°C. To halt the digestion, phenylmethanesulfonyl fluoride was added to each sample, achieving a final concentration of 1 mM and a total volume of 70 μL. A portion of the digested sample (25 μL) was then used for the immunoprecipitation of surrogate and internal standard peptides using triple X proteomics antibodies<span class="citation" data-cites="weiß2018">(<a href="#ref-weiß2018" role="doc-biblioref">14</a>)</span>. These peptides were subsequently analyzed and quantified through LC-MS. The specific surrogate peptides employed for protein quantification and the LC-gradients are detailed in the supplementary material. For data processing, Skyline software (MACOSS Lab, University of Washington, Seattle, WA) and TraceFinder 4.1 (Thermo Fisher Scientific) were utilized.</p>
</section>
</section>
<section id="results" class="level2">
<h2 class="anchored" data-anchor-id="results">Results</h2>
<section id="viability-and-morphology" class="level3">
<h3 class="anchored" data-anchor-id="viability-and-morphology">Viability and morphology</h3>
<p>in <a href="#fig-viability-main" class="quarto-xref">Figure&nbsp;1</a> we see that a relatively stable viability during the first 9 days of culture, and stayed above 60 percent of the variability after spheroids formation. Viability for donor was stable throughout the experiment, for the timepoints measured. Donor 12 was stable for the first 9 days of the experiment, with a slight dropoff seen from day 12, but stable hereafter., never dropping below 60 % of the viability after spheroid formation. Donor 3 was stable for the first 12 days, with a slight dropoff hereafter. Furthermore, morpholgy (<a href="#fig-morph" class="quarto-xref">Figure&nbsp;2</a>) was consistent throughout the experiment, disregardign a tendency for the spheroids to shrink during the experimentm, which might be due to compaction or shedding of cells?(main speroid article had atp relative to size).</p>
<p>Some loss of viability or metabolic activity cannot be ruled out, however this should not be a major concern since we are looking at relative data within each timepoints.</p>
<p>Within nine days from seeding the spheroid has formed and exhibit a tightly compacted appearance.</p>
<div id="fig-viability-main" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-viability-main-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="notebooks/viability/output/main_viability.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1: Each point is the mean ATP content of 5-8 spheroids/replicates (mean = 7). Missing data for for donor 1 on day 5 and 16. Data for donor 3 day 9 was measured on day 10"><img src="notebooks/viability/output/main_viability.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-viability-main-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Each point is the mean ATP content of 5-8 spheroids/replicates (mean = 7). Missing data for for donor 1 on day 5 and 16. Data for donor 3 day 9 was measured on day 10
</figcaption>
</figure>
</div>
<div id="fig-morph" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-morph-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/morph.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;2: Morphology for donors one, two, and three. Photos from the same well for each donor throughout the experiment. Scale bars represent 200 µm."><img src="images/morph.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-morph-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Morphology for donors one, two, and three. Photos from the same well for each donor throughout the experiment. Scale bars represent 200 µm.
</figcaption>
</figure>
</div>
</section>
<section id="effect-of-inflammatory-cytokines-on-mrna-expression" class="level3">
<h3 class="anchored" data-anchor-id="effect-of-inflammatory-cytokines-on-mrna-expression">Effect of inflammatory cytokines on mRNA expression</h3>
<p>Employing the 3D model and more physiologically relevant concentrations of the pro-inflammatory cytokines IL-6 and IL-1β, we observed a dose-dependent downregulation of the transcription of the critical CYP3A4 enzyme, also at physiologically relevant concentrations, a notable finding that deviates from much of the existing literature (hører til I discussion, snarere end her?). Similar dose-dependent effects were seen with CYP1A2, 2B6, and 2D6 upon cytokine exposure, except for an unexpected upregulation of 2B6 after treatment with IL1-B.</p>
<p>No negative effect of the treatments on viability or morphology was observed (rather the reverse, likely due to the excessive handling of the plates for viability/morphology). (Har også viability for treatments. Plottes og tilføjes supp.?)</p>
<div class="quarto-embed-nb-cell" data-notebook="/home/runner/work/inflammation_spheroids/inflammation_spheroids/notebooks/qPCR/mRNA.qmd" data-notebook-title="mRNA" data-notebook-cellid="cell-fig-mRNA-CYP3A4-main">
<div class="cell-container"><div class="cell-decorator"><pre>In [2]:</pre></div><div id="cell-fig-mRNA-CYP3A4-main" class="cell" data-fig-width="9.5" data-fig-height="6.5">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Subset the data for each condition</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>data_IL1B <span class="op">&lt;-</span> data_3A4[condition <span class="op">==</span> <span class="st">"IL-1B"</span>]</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>data_IL1B_means <span class="op">&lt;-</span> overall_means[condition <span class="op">==</span> <span class="st">"IL-1B"</span>]</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>data_IL6 <span class="op">&lt;-</span> data_3A4[condition <span class="op">==</span> <span class="st">"IL-6"</span>]</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>data_IL6_means <span class="op">&lt;-</span> overall_means[condition <span class="op">==</span> <span class="st">"IL-6"</span>]</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Split the treatment string so we can have a continous x axis. Otherwise we cannot put lines or rectangles at relevant levels of ILs</span></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>data_IL1B[, conc <span class="op">:=</span> tstrsplit(treatment, <span class="st">" "</span>, <span class="bu">type</span>.convert <span class="op">=</span> TRUE, keep <span class="op">=</span> <span class="dv">1</span>)]</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>data_IL1B_means[, conc <span class="op">:=</span> tstrsplit(treatment, <span class="st">" "</span>, <span class="bu">type</span>.convert <span class="op">=</span> TRUE, keep <span class="op">=</span> <span class="dv">1</span>)]</span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>data_IL6[, conc <span class="op">:=</span> tstrsplit(treatment, <span class="st">" "</span>, <span class="bu">type</span>.convert <span class="op">=</span> TRUE, keep <span class="op">=</span> <span class="dv">1</span>)]</span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>data_IL6_means[, conc <span class="op">:=</span> tstrsplit(treatment, <span class="st">" "</span>, <span class="bu">type</span>.convert <span class="op">=</span> TRUE, keep <span class="op">=</span> <span class="dv">1</span>)]</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate the plot for IL-1B condition</span></span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>plot_IL1B <span class="op">&lt;-</span></span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>  ggplot(</span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>    data_IL1B,</span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a>    aes(</span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a>      x <span class="op">=</span> conc,</span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a>      y <span class="op">=</span> time_means</span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb5-21"><a href="#cb5-21" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-22"><a href="#cb5-22" aria-hidden="true" tabindex="-1"></a>  <span class="co">## First layer ##</span></span>
<span id="cb5-23"><a href="#cb5-23" aria-hidden="true" tabindex="-1"></a>  geom_point(</span>
<span id="cb5-24"><a href="#cb5-24" aria-hidden="true" tabindex="-1"></a>    aes(</span>
<span id="cb5-25"><a href="#cb5-25" aria-hidden="true" tabindex="-1"></a>      fill <span class="op">=</span> factor(time)</span>
<span id="cb5-26"><a href="#cb5-26" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb5-27"><a href="#cb5-27" aria-hidden="true" tabindex="-1"></a>    size <span class="op">=</span> <span class="dv">7</span>,</span>
<span id="cb5-28"><a href="#cb5-28" aria-hidden="true" tabindex="-1"></a>    shape <span class="op">=</span> <span class="dv">21</span>,  <span class="co"># Use a shape that is filled, 21 is filled</span></span>
<span id="cb5-29"><a href="#cb5-29" aria-hidden="true" tabindex="-1"></a>    color <span class="op">=</span> <span class="st">"black"</span>,</span>
<span id="cb5-30"><a href="#cb5-30" aria-hidden="true" tabindex="-1"></a>    stroke <span class="op">=</span> <span class="dv">1</span>,</span>
<span id="cb5-31"><a href="#cb5-31" aria-hidden="true" tabindex="-1"></a>    alpha <span class="op">=</span> <span class="fl">.7</span></span>
<span id="cb5-32"><a href="#cb5-32" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-33"><a href="#cb5-33" aria-hidden="true" tabindex="-1"></a>  scale_fill_brewer(palette <span class="op">=</span> <span class="st">"Set2"</span>) <span class="op">+</span></span>
<span id="cb5-34"><a href="#cb5-34" aria-hidden="true" tabindex="-1"></a>  <span class="co">## Second layer ##</span></span>
<span id="cb5-35"><a href="#cb5-35" aria-hidden="true" tabindex="-1"></a>  geom_point(</span>
<span id="cb5-36"><a href="#cb5-36" aria-hidden="true" tabindex="-1"></a>    data <span class="op">=</span> data_IL1B_means,</span>
<span id="cb5-37"><a href="#cb5-37" aria-hidden="true" tabindex="-1"></a>    aes(</span>
<span id="cb5-38"><a href="#cb5-38" aria-hidden="true" tabindex="-1"></a>      x <span class="op">=</span> conc,</span>
<span id="cb5-39"><a href="#cb5-39" aria-hidden="true" tabindex="-1"></a>      y <span class="op">=</span> overall_mean</span>
<span id="cb5-40"><a href="#cb5-40" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb5-41"><a href="#cb5-41" aria-hidden="true" tabindex="-1"></a>    size <span class="op">=</span> <span class="dv">12</span>, alpha <span class="op">=</span> <span class="fl">0.80</span></span>
<span id="cb5-42"><a href="#cb5-42" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-43"><a href="#cb5-43" aria-hidden="true" tabindex="-1"></a>  <span class="co">## Modifications ##</span></span>
<span id="cb5-44"><a href="#cb5-44" aria-hidden="true" tabindex="-1"></a>  labs(</span>
<span id="cb5-45"><a href="#cb5-45" aria-hidden="true" tabindex="-1"></a>    x <span class="op">=</span> <span class="st">"Treatment"</span>,</span>
<span id="cb5-46"><a href="#cb5-46" aria-hidden="true" tabindex="-1"></a>    y <span class="op">=</span> <span class="st">"Expression relative to vehicle control"</span>,</span>
<span id="cb5-47"><a href="#cb5-47" aria-hidden="true" tabindex="-1"></a>    fill <span class="op">=</span> <span class="st">"Duration of treatment, hours:"</span></span>
<span id="cb5-48"><a href="#cb5-48" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-49"><a href="#cb5-49" aria-hidden="true" tabindex="-1"></a>  scale_y_continuous(limits <span class="op">=</span> c(<span class="dv">0</span>, <span class="fl">1.25</span>), n.breaks <span class="op">=</span> <span class="dv">6</span>) <span class="op">+</span></span>
<span id="cb5-50"><a href="#cb5-50" aria-hidden="true" tabindex="-1"></a>  scale_x_log10(</span>
<span id="cb5-51"><a href="#cb5-51" aria-hidden="true" tabindex="-1"></a>    name <span class="op">=</span> <span class="st">"Concentration IL-1β (ng/ml)"</span>) <span class="op">+</span></span>
<span id="cb5-52"><a href="#cb5-52" aria-hidden="true" tabindex="-1"></a>  annotation_logticks(sides <span class="op">=</span> <span class="st">"b"</span>)  <span class="op">+</span></span>
<span id="cb5-53"><a href="#cb5-53" aria-hidden="true" tabindex="-1"></a>  geom_hline(yintercept <span class="op">=</span> <span class="dv">1</span>, linetype <span class="op">=</span> <span class="st">"dashed"</span>) <span class="op">+</span></span>
<span id="cb5-54"><a href="#cb5-54" aria-hidden="true" tabindex="-1"></a>  theme_bw() <span class="op">+</span></span>
<span id="cb5-55"><a href="#cb5-55" aria-hidden="true" tabindex="-1"></a>  theme(</span>
<span id="cb5-56"><a href="#cb5-56" aria-hidden="true" tabindex="-1"></a>    <span class="co">#axis.text.x = element_text(angle = 30, hjust = 1),</span></span>
<span id="cb5-57"><a href="#cb5-57" aria-hidden="true" tabindex="-1"></a>    axis.text <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">12</span>, face <span class="op">=</span> <span class="st">"bold"</span>),</span>
<span id="cb5-58"><a href="#cb5-58" aria-hidden="true" tabindex="-1"></a>    axis.title.y <span class="op">=</span> element_text(vjust <span class="op">=</span> <span class="dv">3</span>,</span>
<span id="cb5-59"><a href="#cb5-59" aria-hidden="true" tabindex="-1"></a>                                face <span class="op">=</span> <span class="st">"bold"</span>,</span>
<span id="cb5-60"><a href="#cb5-60" aria-hidden="true" tabindex="-1"></a>                                size <span class="op">=</span> <span class="dv">15</span>,</span>
<span id="cb5-61"><a href="#cb5-61" aria-hidden="true" tabindex="-1"></a>                                margin <span class="op">=</span> margin(t <span class="op">=</span> <span class="dv">0</span>, r <span class="op">=</span> <span class="dv">10</span>, b <span class="op">=</span> <span class="dv">0</span>, l <span class="op">=</span> <span class="dv">0</span>)),</span>
<span id="cb5-62"><a href="#cb5-62" aria-hidden="true" tabindex="-1"></a>    plot.margin <span class="op">=</span> margin(t <span class="op">=</span> <span class="dv">10</span>, r <span class="op">=</span> <span class="dv">10</span>, b <span class="op">=</span> <span class="dv">10</span>, l <span class="op">=</span> <span class="dv">30</span>),</span>
<span id="cb5-63"><a href="#cb5-63" aria-hidden="true" tabindex="-1"></a>    axis.title.x <span class="op">=</span> element_text(vjust <span class="op">=</span> <span class="op">-</span><span class="fl">1.5</span>, face <span class="op">=</span> <span class="st">"bold"</span>, size <span class="op">=</span> <span class="dv">15</span>),</span>
<span id="cb5-64"><a href="#cb5-64" aria-hidden="true" tabindex="-1"></a>    plot.title <span class="op">=</span> element_text(face <span class="op">=</span> <span class="st">"bold"</span>),</span>
<span id="cb5-65"><a href="#cb5-65" aria-hidden="true" tabindex="-1"></a>    <span class="co">#legend.title = element_text(face = "bold"),</span></span>
<span id="cb5-66"><a href="#cb5-66" aria-hidden="true" tabindex="-1"></a>    legend.position <span class="op">=</span> <span class="st">"bottom"</span>,</span>
<span id="cb5-67"><a href="#cb5-67" aria-hidden="true" tabindex="-1"></a>    panel.grid.major.x <span class="op">=</span> element_blank(),</span>
<span id="cb5-68"><a href="#cb5-68" aria-hidden="true" tabindex="-1"></a>    panel.grid.minor.x <span class="op">=</span> element_blank()</span>
<span id="cb5-69"><a href="#cb5-69" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb5-70"><a href="#cb5-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-71"><a href="#cb5-71" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-72"><a href="#cb5-72" aria-hidden="true" tabindex="-1"></a><span class="co"># plot_IL1B</span></span>
<span id="cb5-73"><a href="#cb5-73" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-74"><a href="#cb5-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-75"><a href="#cb5-75" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate the plot for IL-6 condition</span></span>
<span id="cb5-76"><a href="#cb5-76" aria-hidden="true" tabindex="-1"></a>plot_IL6 <span class="op">&lt;-</span></span>
<span id="cb5-77"><a href="#cb5-77" aria-hidden="true" tabindex="-1"></a>  ggplot(</span>
<span id="cb5-78"><a href="#cb5-78" aria-hidden="true" tabindex="-1"></a>    data_IL6,</span>
<span id="cb5-79"><a href="#cb5-79" aria-hidden="true" tabindex="-1"></a>    aes(</span>
<span id="cb5-80"><a href="#cb5-80" aria-hidden="true" tabindex="-1"></a>      x <span class="op">=</span> conc,</span>
<span id="cb5-81"><a href="#cb5-81" aria-hidden="true" tabindex="-1"></a>      y <span class="op">=</span> time_means</span>
<span id="cb5-82"><a href="#cb5-82" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb5-83"><a href="#cb5-83" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-84"><a href="#cb5-84" aria-hidden="true" tabindex="-1"></a>  geom_rect(xmin <span class="op">=</span> log10(<span class="fl">0.00000000001</span>),  <span class="co">#Rect to indicate phys levels. Call before main layes so it does not cover these. </span></span>
<span id="cb5-85"><a href="#cb5-85" aria-hidden="true" tabindex="-1"></a>            xmax <span class="op">=</span> log10(<span class="fl">0.02</span>),</span>
<span id="cb5-86"><a href="#cb5-86" aria-hidden="true" tabindex="-1"></a>            ymin <span class="op">=</span> <span class="op">-</span>Inf,</span>
<span id="cb5-87"><a href="#cb5-87" aria-hidden="true" tabindex="-1"></a>            ymax <span class="op">=</span> Inf,</span>
<span id="cb5-88"><a href="#cb5-88" aria-hidden="true" tabindex="-1"></a>            fill <span class="op">=</span> <span class="st">"lightgreen"</span>,</span>
<span id="cb5-89"><a href="#cb5-89" aria-hidden="true" tabindex="-1"></a>            alpha <span class="op">=</span> <span class="fl">0.006</span></span>
<span id="cb5-90"><a href="#cb5-90" aria-hidden="true" tabindex="-1"></a>            ) <span class="op">+</span></span>
<span id="cb5-91"><a href="#cb5-91" aria-hidden="true" tabindex="-1"></a>  geom_rect(xmin <span class="op">=</span> log10(<span class="fl">0.015</span>),  <span class="co">#Rect to indicate phys levels. Call before main layes so it does not cover these. </span></span>
<span id="cb5-92"><a href="#cb5-92" aria-hidden="true" tabindex="-1"></a>            xmax <span class="op">=</span> log10(<span class="fl">0.15</span>),</span>
<span id="cb5-93"><a href="#cb5-93" aria-hidden="true" tabindex="-1"></a>            ymin <span class="op">=</span> <span class="op">-</span>Inf,</span>
<span id="cb5-94"><a href="#cb5-94" aria-hidden="true" tabindex="-1"></a>            ymax <span class="op">=</span> Inf,</span>
<span id="cb5-95"><a href="#cb5-95" aria-hidden="true" tabindex="-1"></a>            fill <span class="op">=</span> <span class="st">"yellow"</span>,</span>
<span id="cb5-96"><a href="#cb5-96" aria-hidden="true" tabindex="-1"></a>            alpha <span class="op">=</span> <span class="fl">0.006</span></span>
<span id="cb5-97"><a href="#cb5-97" aria-hidden="true" tabindex="-1"></a>            ) <span class="op">+</span></span>
<span id="cb5-98"><a href="#cb5-98" aria-hidden="true" tabindex="-1"></a>  geom_rect(xmin <span class="op">=</span> log10(<span class="fl">0.09</span>),  <span class="co">#Rect to indicate phys levels. Call before main layes so it does not cover these. </span></span>
<span id="cb5-99"><a href="#cb5-99" aria-hidden="true" tabindex="-1"></a>            xmax <span class="op">=</span> log10(Inf),</span>
<span id="cb5-100"><a href="#cb5-100" aria-hidden="true" tabindex="-1"></a>            ymin <span class="op">=</span> <span class="op">-</span>Inf,</span>
<span id="cb5-101"><a href="#cb5-101" aria-hidden="true" tabindex="-1"></a>            ymax <span class="op">=</span> Inf,</span>
<span id="cb5-102"><a href="#cb5-102" aria-hidden="true" tabindex="-1"></a>            fill <span class="op">=</span> <span class="st">"coral"</span>,</span>
<span id="cb5-103"><a href="#cb5-103" aria-hidden="true" tabindex="-1"></a>            alpha <span class="op">=</span> <span class="fl">0.006</span></span>
<span id="cb5-104"><a href="#cb5-104" aria-hidden="true" tabindex="-1"></a>            ) <span class="op">+</span></span>
<span id="cb5-105"><a href="#cb5-105" aria-hidden="true" tabindex="-1"></a>  <span class="co">## First layer ##</span></span>
<span id="cb5-106"><a href="#cb5-106" aria-hidden="true" tabindex="-1"></a>  geom_point(</span>
<span id="cb5-107"><a href="#cb5-107" aria-hidden="true" tabindex="-1"></a>    aes(</span>
<span id="cb5-108"><a href="#cb5-108" aria-hidden="true" tabindex="-1"></a>      fill <span class="op">=</span> factor(time)</span>
<span id="cb5-109"><a href="#cb5-109" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb5-110"><a href="#cb5-110" aria-hidden="true" tabindex="-1"></a>    size <span class="op">=</span> <span class="dv">7</span>,</span>
<span id="cb5-111"><a href="#cb5-111" aria-hidden="true" tabindex="-1"></a>    shape <span class="op">=</span> <span class="dv">21</span>,  <span class="co"># Use a shape that is filled, 21 is filled</span></span>
<span id="cb5-112"><a href="#cb5-112" aria-hidden="true" tabindex="-1"></a>    color <span class="op">=</span> <span class="st">"black"</span>,</span>
<span id="cb5-113"><a href="#cb5-113" aria-hidden="true" tabindex="-1"></a>    stroke <span class="op">=</span> <span class="dv">1</span>,</span>
<span id="cb5-114"><a href="#cb5-114" aria-hidden="true" tabindex="-1"></a>    alpha <span class="op">=</span> <span class="fl">.7</span></span>
<span id="cb5-115"><a href="#cb5-115" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-116"><a href="#cb5-116" aria-hidden="true" tabindex="-1"></a>  scale_fill_brewer(palette <span class="op">=</span> <span class="st">"Set2"</span>) <span class="op">+</span></span>
<span id="cb5-117"><a href="#cb5-117" aria-hidden="true" tabindex="-1"></a>  <span class="co">## Second layer ##</span></span>
<span id="cb5-118"><a href="#cb5-118" aria-hidden="true" tabindex="-1"></a>  geom_point(</span>
<span id="cb5-119"><a href="#cb5-119" aria-hidden="true" tabindex="-1"></a>    data <span class="op">=</span> data_IL6_means,</span>
<span id="cb5-120"><a href="#cb5-120" aria-hidden="true" tabindex="-1"></a>    aes(</span>
<span id="cb5-121"><a href="#cb5-121" aria-hidden="true" tabindex="-1"></a>      x <span class="op">=</span> conc,</span>
<span id="cb5-122"><a href="#cb5-122" aria-hidden="true" tabindex="-1"></a>      y <span class="op">=</span> overall_mean</span>
<span id="cb5-123"><a href="#cb5-123" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb5-124"><a href="#cb5-124" aria-hidden="true" tabindex="-1"></a>    size <span class="op">=</span> <span class="dv">12</span>, alpha <span class="op">=</span> <span class="fl">0.8</span></span>
<span id="cb5-125"><a href="#cb5-125" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-126"><a href="#cb5-126" aria-hidden="true" tabindex="-1"></a>  <span class="co">## Modifications ##</span></span>
<span id="cb5-127"><a href="#cb5-127" aria-hidden="true" tabindex="-1"></a>  labs(</span>
<span id="cb5-128"><a href="#cb5-128" aria-hidden="true" tabindex="-1"></a>    x <span class="op">=</span> <span class="st">"Treatment"</span>,</span>
<span id="cb5-129"><a href="#cb5-129" aria-hidden="true" tabindex="-1"></a>    y <span class="op">=</span> <span class="st">"Expression relative to vehicle control"</span>,</span>
<span id="cb5-130"><a href="#cb5-130" aria-hidden="true" tabindex="-1"></a>    fill <span class="op">=</span> <span class="st">"Duration of treatment, hours:"</span></span>
<span id="cb5-131"><a href="#cb5-131" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">+</span></span>
<span id="cb5-132"><a href="#cb5-132" aria-hidden="true" tabindex="-1"></a>  scale_y_continuous(limits <span class="op">=</span> c(<span class="dv">0</span>, <span class="fl">1.25</span>), n.breaks <span class="op">=</span> <span class="dv">6</span>) <span class="op">+</span></span>
<span id="cb5-133"><a href="#cb5-133" aria-hidden="true" tabindex="-1"></a>  scale_x_log10(name <span class="op">=</span> <span class="st">"Concentration IL-6 (ng/ml)"</span>) <span class="op">+</span></span>
<span id="cb5-134"><a href="#cb5-134" aria-hidden="true" tabindex="-1"></a>  annotation_logticks(sides <span class="op">=</span> <span class="st">"b"</span>)  <span class="op">+</span></span>
<span id="cb5-135"><a href="#cb5-135" aria-hidden="true" tabindex="-1"></a>  geom_hline(yintercept <span class="op">=</span> <span class="dv">1</span>, linetype <span class="op">=</span> <span class="st">"dashed"</span>) <span class="op">+</span></span>
<span id="cb5-136"><a href="#cb5-136" aria-hidden="true" tabindex="-1"></a>  theme_bw() <span class="op">+</span></span>
<span id="cb5-137"><a href="#cb5-137" aria-hidden="true" tabindex="-1"></a>  theme(</span>
<span id="cb5-138"><a href="#cb5-138" aria-hidden="true" tabindex="-1"></a>    <span class="co">#axis.text.x = element_text(angle = 30, hjust = 1),</span></span>
<span id="cb5-139"><a href="#cb5-139" aria-hidden="true" tabindex="-1"></a>    axis.text <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">12</span>, face <span class="op">=</span> <span class="st">"bold"</span>),</span>
<span id="cb5-140"><a href="#cb5-140" aria-hidden="true" tabindex="-1"></a>    axis.title.y <span class="op">=</span> element_text(vjust <span class="op">=</span> <span class="dv">3</span>,</span>
<span id="cb5-141"><a href="#cb5-141" aria-hidden="true" tabindex="-1"></a>                                face <span class="op">=</span> <span class="st">"bold"</span>,</span>
<span id="cb5-142"><a href="#cb5-142" aria-hidden="true" tabindex="-1"></a>                                size <span class="op">=</span> <span class="dv">20</span>,</span>
<span id="cb5-143"><a href="#cb5-143" aria-hidden="true" tabindex="-1"></a>                                margin <span class="op">=</span> margin(t <span class="op">=</span> <span class="dv">0</span>, r <span class="op">=</span> <span class="dv">10</span>, b <span class="op">=</span> <span class="dv">0</span>, l <span class="op">=</span> <span class="dv">0</span>)),</span>
<span id="cb5-144"><a href="#cb5-144" aria-hidden="true" tabindex="-1"></a>    axis.title.x <span class="op">=</span> element_text(vjust <span class="op">=</span> <span class="op">-</span><span class="fl">1.5</span>, face <span class="op">=</span> <span class="st">"bold"</span>, size <span class="op">=</span> <span class="dv">15</span>),</span>
<span id="cb5-145"><a href="#cb5-145" aria-hidden="true" tabindex="-1"></a>    plot.title <span class="op">=</span> element_text(face <span class="op">=</span> <span class="st">"bold"</span>),</span>
<span id="cb5-146"><a href="#cb5-146" aria-hidden="true" tabindex="-1"></a>    <span class="co">#legend.title = element_text(face = "bold"),</span></span>
<span id="cb5-147"><a href="#cb5-147" aria-hidden="true" tabindex="-1"></a>    legend.position <span class="op">=</span> <span class="st">"bottom"</span>,</span>
<span id="cb5-148"><a href="#cb5-148" aria-hidden="true" tabindex="-1"></a>    plot.margin <span class="op">=</span> margin(t <span class="op">=</span> <span class="dv">10</span>, r <span class="op">=</span> <span class="dv">10</span>, b <span class="op">=</span> <span class="dv">10</span>, l <span class="op">=</span> <span class="dv">30</span>),</span>
<span id="cb5-149"><a href="#cb5-149" aria-hidden="true" tabindex="-1"></a>    panel.grid.major.x <span class="op">=</span> element_blank(),</span>
<span id="cb5-150"><a href="#cb5-150" aria-hidden="true" tabindex="-1"></a>    panel.grid.minor.x <span class="op">=</span> element_blank()</span>
<span id="cb5-151"><a href="#cb5-151" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb5-152"><a href="#cb5-152" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-153"><a href="#cb5-153" aria-hidden="true" tabindex="-1"></a><span class="co"># plot_IL6</span></span>
<span id="cb5-154"><a href="#cb5-154" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-155"><a href="#cb5-155" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-156"><a href="#cb5-156" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-157"><a href="#cb5-157" aria-hidden="true" tabindex="-1"></a><span class="co">#### Combine the plots</span></span>
<span id="cb5-158"><a href="#cb5-158" aria-hidden="true" tabindex="-1"></a><span class="co"># Update individual plots for combination</span></span>
<span id="cb5-159"><a href="#cb5-159" aria-hidden="true" tabindex="-1"></a><span class="co"># Remove the y-axis title from the second plot</span></span>
<span id="cb5-160"><a href="#cb5-160" aria-hidden="true" tabindex="-1"></a>plot_IL6 <span class="op">&lt;-</span> plot_IL6 <span class="op">+</span> theme(axis.title.y <span class="op">=</span> element_blank(),</span>
<span id="cb5-161"><a href="#cb5-161" aria-hidden="true" tabindex="-1"></a>                             <span class="co">#axis.title.x = element_blank(),</span></span>
<span id="cb5-162"><a href="#cb5-162" aria-hidden="true" tabindex="-1"></a>                             ) <span class="op">+</span></span>
<span id="cb5-163"><a href="#cb5-163" aria-hidden="true" tabindex="-1"></a>  ggtitle(<span class="st">"IL-6"</span>)</span>
<span id="cb5-164"><a href="#cb5-164" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-165"><a href="#cb5-165" aria-hidden="true" tabindex="-1"></a>plot_IL1B <span class="op">&lt;-</span> plot_IL1B <span class="op">+</span> </span>
<span id="cb5-166"><a href="#cb5-166" aria-hidden="true" tabindex="-1"></a>  ggtitle(<span class="st">"IL-1β"</span>)</span>
<span id="cb5-167"><a href="#cb5-167" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-168"><a href="#cb5-168" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-169"><a href="#cb5-169" aria-hidden="true" tabindex="-1"></a><span class="co"># Combine the plots with a common legend at the bottom</span></span>
<span id="cb5-170"><a href="#cb5-170" aria-hidden="true" tabindex="-1"></a>combined_plot <span class="op">&lt;-</span> plot_IL1B <span class="op">+</span> plot_IL6 <span class="op">+</span></span>
<span id="cb5-171"><a href="#cb5-171" aria-hidden="true" tabindex="-1"></a>  plot_layout(guides <span class="op">=</span> <span class="st">'collect'</span>) <span class="op">&amp;</span> <span class="co"># This collects and uses a single guide (legend)</span></span>
<span id="cb5-172"><a href="#cb5-172" aria-hidden="true" tabindex="-1"></a>  plot_annotation(</span>
<span id="cb5-173"><a href="#cb5-173" aria-hidden="true" tabindex="-1"></a>    title <span class="op">=</span> <span class="st">'Relative expression of CYP3A4'</span>,</span>
<span id="cb5-174"><a href="#cb5-174" aria-hidden="true" tabindex="-1"></a>    theme <span class="op">=</span> theme(</span>
<span id="cb5-175"><a href="#cb5-175" aria-hidden="true" tabindex="-1"></a>      plot.title <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">22</span>, face <span class="op">=</span> <span class="st">"bold"</span>, hjust <span class="op">=</span> <span class="fl">0.5</span>), <span class="co"># Increase the size of the main title</span></span>
<span id="cb5-176"><a href="#cb5-176" aria-hidden="true" tabindex="-1"></a>      plot.subtitle <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">18</span>) <span class="co"># Increase the size of the subtitles</span></span>
<span id="cb5-177"><a href="#cb5-177" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb5-178"><a href="#cb5-178" aria-hidden="true" tabindex="-1"></a>  ) <span class="op">&amp;</span> </span>
<span id="cb5-179"><a href="#cb5-179" aria-hidden="true" tabindex="-1"></a>  theme(</span>
<span id="cb5-180"><a href="#cb5-180" aria-hidden="true" tabindex="-1"></a>    legend.position <span class="op">=</span> <span class="st">"bottom"</span>,</span>
<span id="cb5-181"><a href="#cb5-181" aria-hidden="true" tabindex="-1"></a>    legend.text <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">14</span>), <span class="co"># Increase the size of the legend text</span></span>
<span id="cb5-182"><a href="#cb5-182" aria-hidden="true" tabindex="-1"></a>    legend.title <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">16</span>) <span class="co"># Increase the size of the legend title</span></span>
<span id="cb5-183"><a href="#cb5-183" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb5-184"><a href="#cb5-184" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-185"><a href="#cb5-185" aria-hidden="true" tabindex="-1"></a><span class="co"># Print the combined plot</span></span>
<span id="cb5-186"><a href="#cb5-186" aria-hidden="true" tabindex="-1"></a>combined_plot</span>
<span id="cb5-187"><a href="#cb5-187" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-188"><a href="#cb5-188" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave(filename = here("notebooks/qPCR/output/mrna_3A4_main.png"), device = "png", dpi = "retina")</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="fig-mrna-cyp3a4-main" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mrna-cyp3a4-main-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/notebooks-qPCR-mRNA-fig-mrna-cyp3a4-main-output-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;3: Relative CYP3A4 mRNA expression. Small points are mean of three donors for 72 and 120 hours, and two donors for 24 hours (donors two and three)"><img src="index_files/figure-html/notebooks-qPCR-mRNA-fig-mrna-cyp3a4-main-output-1.png" width="912" height="624" class="figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mrna-cyp3a4-main-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Relative CYP3A4 mRNA expression. Small points are mean of three donors for 72 and 120 hours, and two donors for 24 hours (donors two and three)
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
</section>
</section>
<section id="references" class="level2">
<h2 class="anchored" data-anchor-id="references">References</h2>
<div id="refs" class="references csl-bib-body" role="list">
<div id="ref-dunvald" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Dunvald ACD, Järvinen E, Mortensen C, Stage TB. Clinical and Molecular Perspectives on Inflammation-Mediated Regulation of Drug Metabolism and Transport. Clinical Pharmacology &amp; Therapeutics [Internet]. n/a(n/a). Available from: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/cpt.2432">https://onlinelibrary.wiley.com/doi/abs/10.1002/cpt.2432</a></div>
</div>
<div id="ref-kim2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Kim HO, Kim HS, Youn JC, Shin EC, Park S. Serum cytokine profiles in healthy young and elderly population assessed using multiplexed bead-based immunoassays. Journal of Translational Medicine [Internet]. 2011 Jul 20;9(1):113. Available from: <a href="https://doi.org/10.1186/1479-5876-9-113">https://doi.org/10.1186/1479-5876-9-113</a></div>
</div>
<div id="ref-kleiner2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Kleiner G, Marcuzzi A, Zanin V, Monasta L, Zauli G. Cytokine Levels in the Serum of Healthy Subjects. Mediators of Inflammation [Internet]. 2013;2013:1–6. Available from: <a href="http://www.hindawi.com/journals/mi/2013/434010/">http://www.hindawi.com/journals/mi/2013/434010/</a></div>
</div>
<div id="ref-said2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Said EA, Al-Reesi I, Al-Shizawi N, Jaju S, Al-Balushi MS, Koh CY, et al. Defining IL-6 levels in healthy individuals: A meta-analysis. Journal of Medical Virology [Internet]. 2021;93(6):3915–24. Available from: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.26654">https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.26654</a></div>
</div>
<div id="ref-strand2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Strand V, Boklage SH, Kimura T, Joly F, Boyapati A, Msihid J. High levels of interleukin-6 in patients with rheumatoid arthritis are associated with greater improvements in health-related quality of life for sarilumab compared with adalimumab. Arthritis Research &amp; Therapy [Internet]. 2020 Oct 20;22(1):250. Available from: <a href="https://doi.org/10.1186/s13075-020-02344-3">https://doi.org/10.1186/s13075-020-02344-3</a></div>
</div>
<div id="ref-umare2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Umare V, Pradhan V, Nadkar M, Rajadhyaksha A, Patwardhan M, Ghosh KK, et al. Effect of Proinflammatory Cytokines (IL-6, TNF-<span><em>α</em></span>, and IL-1<span><em>β</em></span>) on Clinical Manifestations in Indian SLE Patients. Mediators of Inflammation [Internet]. 2014 Dec 7;2014:e385297. Available from: <a href="https://www.hindawi.com/journals/mi/2014/385297/">https://www.hindawi.com/journals/mi/2014/385297/</a></div>
</div>
<div id="ref-franco2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Franco DM, Arevalo-Rodriguez I, Figuls MRi, Oleas NGM, Nuvials X, Zamora J. Plasma interleukin<span>-</span>6 concentration for the diagnosis of sepsis in critically ill adults. Cochrane Database of Systematic Reviews [Internet]. 2019;(4). Available from: <a href="https://www.cochranelibrary.com/cdsr/doi/10.1002/14651858.CD011811.pub2/full">https://www.cochranelibrary.com/cdsr/doi/10.1002/14651858.CD011811.pub2/full</a></div>
</div>
<div id="ref-bell2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Bell CC, Hendriks DFG, Moro SML, Ellis E, Walsh J, Renblom A, et al. <a href="https://doi.org/10.1038/srep25187">Characterization of primary human hepatocyte spheroids as a model system for drug-induced liver injury, liver function and disease</a>. Scientific Reports. 2016 May;6:25187. </div>
</div>
<div id="ref-bell2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Bell CC, Dankers ACA, Lauschke VM, Sison-Young R, Jenkins R, Rowe C, et al. <a href="https://doi.org/10.1093/toxsci/kfx289">Comparison of Hepatic 2D Sandwich Cultures and 3D Spheroids for Long-term Toxicity Applications: A Multicenter Study</a>. Toxicological Sciences: An Official Journal of the Society of Toxicology. 2018 Apr 1;162(2):655–66. </div>
</div>
<div id="ref-järvinen2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Järvinen E, Hammer HS, Pötz O, Ingelman-Sundberg M, Stage TB. 3D Spheroid Primary Human Hepatocytes for Prediction of Cytochrome P450 and Drug Transporter Induction. Clinical Pharmacology &amp; Therapeutics [Internet]. 2023;113(6):1284–94. Available from: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/cpt.2887">https://onlinelibrary.wiley.com/doi/abs/10.1002/cpt.2887</a></div>
</div>
<div id="ref-ingelman-sundberg" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Ingelman-Sundberg M, Lauschke VM. 3D human liver spheroids for translational pharmacology and toxicology. Basic &amp; Clinical Pharmacology &amp; Toxicology [Internet]. n/a(n/a). Available from: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/bcpt.13587">https://onlinelibrary.wiley.com/doi/abs/10.1111/bcpt.13587</a></div>
</div>
<div id="ref-toni2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Toni LS, Garcia AM, Jeffrey DA, Jiang X, Stauffer BL, Miyamoto SD, et al. Optimization of phenol-chloroform RNA extraction. MethodsX [Internet]. 2018 Jan 1;5:599–608. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S2215016118300773">https://www.sciencedirect.com/science/article/pii/S2215016118300773</a></div>
</div>
<div id="ref-derveaux2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Derveaux S, Vandesompele J, Hellemans J. How to do successful gene expression analysis using real-time PCR. Methods [Internet]. 2010 Apr 1;50(4):227–30. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S1046202309002461">https://www.sciencedirect.com/science/article/pii/S1046202309002461</a></div>
</div>
<div id="ref-weiß2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Weiß F, Hammer HS, Klein K, Planatscher H, Zanger UM, Norén A, et al. <a href="https://doi.org/10.1124/dmd.117.078626">Direct Quantification of Cytochromes P450 and Drug Transporters-A Rapid, Targeted Mass Spectrometry-Based Immunoassay Panel for Tissues and Cell Culture Lysates</a>. Drug Metabolism and Disposition: The Biological Fate of Chemicals. 2018 Apr;46(4):387–96. </div>
</div>
<div id="ref-savaryn2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Savaryn JP, Liu N, Sun J, Ma J, Stresser DM, Jenkins G. Enrichment-free High-throughput Liquid Chromatography<span></span>Multiple-Reaction Monitoring Quantification of Cytochrome P450 Proteins in Plated Human Hepatocytes Direct from 96-Well Plates Enables Routine Protein Induction Measurements. Drug Metabolism and Disposition [Internet]. 2020 Jul 1;48(7):594–602. Available from: <a href="https://dmd.aspetjournals.org/content/48/7/594">https://dmd.aspetjournals.org/content/48/7/594</a></div>
</div>
</div>
</section>

     </main>
<!-- /main column -->  <script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>  </div> <!-- /content -->  <script>var lightboxQuarto = GLightbox({"descPosition":"bottom","openEffect":"zoom","selector":".lightbox","loop":false,"closeEffect":"zoom"});
window.onload = () => {
  lightboxQuarto.on('slide_before_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    const href = trigger.getAttribute('href');
    if (href !== null) {
      const imgEl = window.document.querySelector(`a[href="${href}"] img`);
      if (imgEl !== null) {
        const srcAttr = imgEl.getAttribute("src");
        if (srcAttr && srcAttr.startsWith("data:")) {
          slideConfig.href = srcAttr;
        }
      }
    } 
  });

  lightboxQuarto.on('slide_after_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    if (window.Quarto?.typesetMath) {
      window.Quarto.typesetMath(slideNode);
    }
  });

};
          </script> 
  
</body></html>